community_tasks/french_evals.py

```diff
@@ -80,6 +80,18 @@ def prompt_gpqa_fr(line, task_name: str = None):
     )
 
 
+# bac-fr prompt function
+def prompt_bac_fr(line, task_name: str = None):
+    return Doc(
+        task_name=task_name,
+        query=line["prompt"],
+        choices=[""],
+        gold_index=0,
+        instruction="",
+        specific={"instruction": line["instruction"], "enonce": line["enonce"]},
```

Comment:
Why do you put instruction in specific? Aren't we using the same instruction for all tasks there (or maybe all tasks, per domain)?